tests/test-sessions/test-hazelcast-sessions/pom.xml

```diff
@@ -85,7 +92,7 @@
     </dependency>
     <dependency>
        <groupId>com.hazelcast</groupId>
-       <artifactId>hazelcast-all</artifactId>
+       <artifactId>hazelcast</artifactId>
```

Comment:
Do we need this dependency here if the `jetty-hazelcast` module already has this dependency?

Reply:
yes. because `jetty-hazelcast` is in `test` scope. And `test` scope is not transitive.